core/templates/app.constants.ts

```diff
@@ -237,5 +237,7 @@ export const AppConstants = {
   },
   CONTRIBUTOR_BADGE_INITIAL_LEVELS: [1, 10, 20, 50, 100, 200, 300, 400, 500],
   CONTRIBUTOR_CERTIFICATE_LOGO:
-    '/assets/images/contributor_dashboard/oppia-logo.jpg'
+    '/assets/images/contributor_dashboard/oppia-logo.jpg',
+
+  ERROT_TOAST_DISPLAY_TIME: 5000
```

Comment:
I would recommend making this a local constant.

Also, suggest changing the name to "TOAST_MESSAGE_DISPLAY_TIME_MSEC"

Reply:
Done